.github/workflows/ci.yaml

```diff
@@ -25,12 +25,13 @@ jobs:
         run: |
           remark -f -u validate-links .
 
-      - name: Validate sample collections
-        run: |
-          cd collection-spec/examples
-          esmcol-validator sample-glade-cmip6-netcdf-collection.json
-          esmcol-validator sample-pangeo-cmip6-collection.json
-          esmcol-validator --esmcol-spec-dirs ../json-schema simple-collection-with-catalog-dict.json
+
+      # - name: Validate sample collections
```

Comment:
You can likely replace it with one of these: https://github.com/m-mohr/stac-node-validator/blob/master/COMPARISON.md